task/upload.go

```diff
@@ -156,7 +156,7 @@ func TaskTranscodeFile(tctx *TaskContext) (*TaskHandlerOutput, error) {
 		tctx:  tctx,
 		inUrl: params.Input.URL,
 		getOutputLocations: func() ([]clients.OutputLocation, error) {
-			_, outputLocation, err := outputLocations(params.Storage.URL, params.Outputs.HLS.Path, false)
+			_, outputLocation, err := outputLocations(params.Storage.URL, params.Outputs.HLS.Path, false, false)
```

Comment:
Good question. I think we want to be able to generate MP4 files with the Transcode API, but I'm not sure if we want to always do it or introduce a flag for it. @ericxtang what do you think?

Reply:
Sure! Still a `staticMp4` flag in the params or would it be better to add a `mp4` output in the outputs fields in the transcode request?

Reply:
Related discussion: https://github.com/livepeer/catalyst-api/pull/518#discussion_r1132710013

IMO we should not have it automatically generated for transcode API unless users explicitly request for it (the automatic generation, that is). It might be cleaner to wait until we have the Catalyst enum for this and then create a corresponding one on the Studio transcode-file API.

Reply:
Yes, we'll have a parameter in Transcode API to generate MP4. Here's the related Linear: https://linear.app/livepeer/project/mp4-in-transcode-api-708bdd704de5